package.json

```diff
@@ -68,6 +68,7 @@
       "./test/_setup-browser-env.js",
       "ignore-styles"
     ],
+    "timeout": "3m",
```

Comment:
This is a guess.  Something changed in AVAs timeout it was working fine on my local machine and not on the CircleCI build.

Reply:
Just checked, in v2. there is no default timeout
https://github.com/avajs/ava/blob/v2.4.0/docs/07-test-timeouts.md#test-timeouts